stdlib/LinearAlgebra/src/lbt.jl

```diff
@@ -13,6 +13,18 @@ struct lbt_library_info_t
     f2c::Int32
     cblas::Int32
 end
+
+macro get_warn(map, key)
+    return quote
+        if !haskey($(esc(map)), $(esc(key)))
```

Comment:
Minor style nitpicking: I personally prefer reading positive conditions if there's an `else` branch anyway, but it's a matter of taste probably.

Reply:
In this case, I like having the "exceptional" circumstance as explicit as possible, as that's the whole point of this method.